internal/proxy/task.go

```diff
@@ -531,6 +523,8 @@ func (dct *describeCollectionTask) Execute(ctx context.Context) error {
 
 	if result.Status.ErrorCode != commonpb.ErrorCode_Success {
```

Comment:
maybe add nil check for `result.Status`

Reply:
this is not related to this PR, will change in another PRs